src_graphql/index_test.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 The Oppia Authors. All Rights Reserved.
```

Comment:
Why does this test file not contain any tests and yet is not in our per-file coverage exclusion list? That suggests to me that it isn't being picked up by our backend test runner

Reply:
For now, I added all of them to the per-file exclusion list but I am planning to add the tests in a follow-up PR( with migrating remaining rest API to graphql ). please let me know if that sounds good to you

Reply:
I think adding them to the exclusion list is fine, but it still doesn't look like your new tests are running. They aren't listed in the [backend coverage report](https://github.com/oppia/oppia/actions/runs/5024501927/jobs/9010965857?pr=18197#step:14:12)

Reply:
I think that is because they don't have any tests to run. If you check previous commits for e.g [this commit](https://github.com/oppia/oppia/pull/18197/commits/7d14158074368adecadedb2453a9f64f35b09794), you can find github_domain in the coverage backend test because some tests were there, which I removed later.